doc/dev/TESTING.md

```diff
@@ -7,8 +7,8 @@ either to validate your build or because you are contributing to F3D and want to
 
 There is a few CMake options to enable when configuring F3D with testing:
 * `BUILD_TESTING`: Enable the tests, enable this one.
-* `F3D_ENABLE_LONG_TIMEOUT_TESTS`: Certain tests can take a bit longer to run, hence they are disabled by default, you may want to enable this one.
-* `F3D_DISABLE_DEFAULT_LIGHTS_TESTS_COMPARISON`: With VTK < 9.0.0, rendering can be very different, although not incorrect, so this option is provided, you may want to keep this one disabled
+* `F3D_TESTING_ENABLE_LONG_TIMEOUT_TESTS`: Certain tests can take a bit longer to run, hence they are disabled by default, you may want to enable this one.
+* `F3D_TESTING_DISABLE_DEFAULT_LIGHTS_TESTS_COMPARISON`: With VTK < 9.0.0, rendering can be very different, although not incorrect, so this option is provided, you may want to keep this one disabled
```

Comment:
Add doc for `F3D_TESTING_ENABLE_RENDERING_TESTS` and `F3D_TESTING_ENABLE_HDRI_TESTS`